### ISSUE TITLE
Swapped width and height

### ISSUE THREAD

**Issue by Hunger-Prevails** [GabPrato/batch-transforms#3]
`Hi Gabriele`, huge respect for what you managed to achieve here.
Your knowledge and expertise about tensor indexing truly impress as previously I couldn't imagine how random crop operations could be done in batches.
And after studying your code for a while I'm gradually coming to understand how such complex indexing works.

But one small thing here on line 123 of batchnorms.py really buffles me:

`w, h = padded.size(2), padded.size(3)`

I can't understand why the height and width variables are swapped. If this is indeed your intention, could you kindly shed some light on why it should be this way?

**Comment by pratogab** [comment]
Hi, just wanted to let you know that I've seen your message. Having a quick look, it does seem like I've accidentally swapped the `w` and `h`. But I would need to deep dive back into the code to make sure. I will do that soon :)

**Comment by Hunger-Prevails** [comment]
> Hi, just wanted to let you know that I've seen your message. Having a quick look, it does seem like I've accidentally swapped the `w` and `h`. But I would need to deep dive back into the code to make sure. I will do that soon :)

Hi, I tried out your implementation with w and h restored to the natural order back when I opened this issue and it functions perfectly to this day. I think you were probably working with square-shaped inputs when you wrote this, therefore this accidental swap didn't pose a problem for you.

Anyway, your implementation saves me a lot of time and allows my code to run quite a bit faster!
